ci/publish-oci.sh

```diff
@@ -70,121 +68,35 @@ function on_exit() {
 
 trap on_exit SIGHUP SIGINT SIGQUIT SIGABRT EXIT
 
-function gen_manifest() {
-  local arch="${1}"
-  local artifact_name="${2}"
-  local artifact_path="${3}"
-  local commands="commands"
-  if [[ "${arch}" =~ "windows" ]]; then
-    artifact_path="${3}.exe"
-  fi
-  if [[ "$artifact_path" =~ .jar ]]; then
-    commands="jar-commands"
-  fi
-  local artifact_desc="${4}"
-  echo '
-apiVersion: digitalasset.com/v1
-kind: Component
-spec:
-  '"${commands}"':
-    - path: '"${artifact_path}"'
-      name: '"${artifact_name}"'
-      desc: '"${artifact_desc}"'
-  '
-}
-
 function publish_artifact {
   local artifact_name="${1}"
-  local artifact_path="${2}"
-  local artifact_desc="${3}"
-  if [[ "$artifact_path" =~ .jar ]]; then
-    declare -a artifact_platforms=( "generic" )
-  else
-    declare -a artifact_platforms=( "linux-arm,linux/arm64" "linux-intel,linux/amd64" "macos,darwin/arm64" "macos,darwin/amd64" "windows,windows/amd64" )
-  fi
+  declare -a artifact_platforms=( "linux-intel,linux/amd64" "linux-arm,linux/arm64" "macos,darwin/arm64" "macos,darwin/amd64" "windows,windows/amd64" )
   declare -a platform_args
-  local artifact arch search_pattern
-
-cd "${STAGING_DIR}" || exit 1
- (
-  # shellcheck disable=SC2068
-  for item in ${artifact_platforms[@]}; do
-    if [[ "$artifact_path" =~ .jar ]]; then
-        arch="${item}"
-        search_pattern="${artifact_path%%/*}"
-      else
-        arch="${item##*,}"
-        search_pattern="${artifact_name}-${RELEASE_TAG}-${item%%,*}.tar.gz"
-    fi
-    info "Processing ${artifact_name} for ${arch}...\n"
-    artifact="$(find . -type f -name ${search_pattern} | head -1)"
-    info "Found artifact: ${artifact}\n"
-    if [[ -f "${artifact}" ]]; then
+  cd "${STAGING_DIR}" || exit 1
+  (
+    for item in ${artifact_platforms[@]}; do
```

Comment:
You can put double quotes around this, as in 
```suggestion
    for item in "${artifact_platforms[@]}"; do
```
When using `@`, the array still gets expanded as multiple arguments, but each argument is quoted to prevent further expansion. https://www.shellcheck.net/wiki/SC2068